src/ray/object_manager/test/create_request_queue_test.cc

```diff
@@ -181,14 +189,15 @@ TEST_F(CreateRequestQueueTest, TestTransientOom) {
   auto req_id1 = queue.AddRequest(ObjectID::Nil(), client, oom_request);
   auto req_id2 = queue.AddRequest(ObjectID::Nil(), client, blocked_request);
 
-  // Transient OOM should not use up any retries.
+  // Transient OOM should happen until the grace period.
   for (int i = 0; i < 3; i++) {
```

Comment:
I think we should advance the clock before doing the retries, to check that we still throw the transient OOM error.